libnavigation-base/src/main/java/com/mapbox/navigation/base/route/dto/BannerComponentsNavigationDto.kt

```diff
@@ -0,0 +1,27 @@
+package com.mapbox.navigation.base.route.dto
+
+import com.google.gson.annotations.SerializedName
+import com.mapbox.navigation.base.route.model.BannerComponentsNavigation
+
+class BannerComponentsNavigationDto(
+    val text: String?,
+    val type: String?,
+    @SerializedName("abbr")
+    val abbreviation: String?,
+    @SerializedName("abbr_priority")
+    val abbreviationPriority: Int?,
+    @SerializedName("imageBaseURL")
+    val imageBaseUrl: String?,
+    val directions: List<String>?,
+    val active: Boolean?
+)
+
+internal fun BannerComponentsNavigationDto.mapToModel() = BannerComponentsNavigation(
```

Comment:
See comment above. Why not using `BannerComponentsNavigation` directly? If so this mapper wouldn't be necessary. Am I missing something?